doc/server/manual-sphinx/userAccessInfoPOST.rst

```diff
@@ -0,0 +1,66 @@
+=========================
+POST-User_access_info
+=========================
+ **Please refer when you Make a New User Access Info.**
+
+Request
+=======
+
+Path
+----
+.. list-table::
+   :header-rows: 1
+
+   * - URL
+     - Comments
+   * - /user/<userId>/access-info
+     - N/A
+
+Parameters
+----------
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+     - Comments
+   * - hostgroupId
+     - N/A
+     - If you select all host groups, the value should be `*`.
```

Comment:
I think that it should be use `must` here instead of `should`.
Because `should` only proposes to users.
But `must` means obligation.